### DISCUSSION TITLE
How to calculate NDVI

### DISCUSSION THREAD

**Discussion by FranklinJS** [gee-community/qgis-earthengine-plugin#95] [Q&A]
Hi

How to calculate NDVI or any other expression?

Thanks

**Comment by giswqs** [comment]
This might help:

- https://github.com/giswqs/qgis-earthengine-examples/blob/master/Image/expressions.py
- https://github.com/giswqs/qgis-earthengine-examples/blob/master/Image/normalized_difference.py

**Reply by FranklinJS** [reply]
Thank you!
